From 9344269f5b940c65fd8701afff4b820ecb033e24 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Thu, 14 Dec 2023 22:13:44 +0000
Subject: [PATCH 1/1] data: Rename to just "User Manager"

As this ulitity is used on XFCE as well as MATE.
---
 data/mate-user-admin.desktop.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/mate-user-admin.desktop.in b/data/mate-user-admin.desktop.in
index 41cd8f6..2131613 100644
--- a/data/mate-user-admin.desktop.in
+++ b/data/mate-user-admin.desktop.in
@@ -1,6 +1,6 @@
 [Desktop Entry]
-Name=MATE User Manager
-Name[ca]=Gestor d'usuaris de MATE
+Name=User Manager
+Name[ca]=Gestor d'usuaris
 Name[zh_CN]=用户管理
 Comment=Manage User 
 Comment[ca]=Gestioneu els usuaris
-- 
2.43.0

